.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish to crates.io
+
+on:
+  push:
+    branches:
+      main
+
+jobs:
+  publish:
+    # need to rename environment name after this is merged
```

Comment:
Why cannot you do this now?

Reply:
Ohhh, secrets, silly me. >.<

Reply:
This needs to be changed in the future as now name of environment stands out - I'll try to follow up with repo admin